stable/redis-cluster/templates/NOTES.txt

```diff
@@ -0,0 +1,12 @@
+Redis cluster can be accessed via port 3306 on the following DNS name from within your cluster:
```

Comment:
I think the port should be `6379`. Same issue in the existing stable `redis` chart but fixed in https://github.com/kubernetes/charts/pull/378/files#diff-2327a4dafbc1e0a885ae92469f932d0c.